core/src/main/python/wlsdeploy/util/variables.py

```diff
@@ -357,15 +357,15 @@ def _init_secret_token_map(model_context):
 
     locations = os.environ.get(_secret_dirs_variable, _secret_dirs_default)
     for dir in locations.split(","):
-         if not os.path.isdir(dir):
```

Comment:
Do we still wants to have the default set to /weblogic-operator/config-overrides-secrets for public use ?  May be we should remove it ?